backend/curation/api/v1/curation/collections/collection_id/actions.py

```diff
@@ -39,7 +43,10 @@ def get(collection_id: str, token_info: dict) -> Response:
 
 
 def patch(collection_id: str, body: dict, token_info: dict) -> Response:
+
     user_info = UserInfo(token_info)
+    if not user_info.is_cxg_admin():
+        raise ForbiddenHTTPException("Unauthorized") from None
```

Comment:
Kick off reprocessing for a whole collection with a `PATCH` to `/curation/v1/collections/{collection_id}`. Works for private and public Collections.